stl/inc/functional

```diff
@@ -1282,29 +1281,22 @@ _NODISCARD bool operator!=(nullptr_t, const function<_Fty>& _Other) noexcept {
 
 // PLACEHOLDERS
 template <int _Nx>
-struct _Ph { // placeholder
-};
+struct _Ph {}; // placeholder

```

Comment:
"Instead of naming this `_Placeholder`, I'll abbreviate it and leave a comment to tell people that I should have named it `_Placeholder`." (No change requested.)

Reply:
This might have been significant back when the compiler complained about long mangled names - placeholders appearing in bind expressions would contribute to the length.